src/main/scala/com/fulcrumgenomics/util/Metric.scala

```diff
@@ -99,56 +99,74 @@ object Metric extends LazyLogging {
     }
   }
 
+  private[util] def build[T <: Metric](reflectiveBuilder: ReflectiveBuilder[T],
+                                       toArg: Int => String,
```

Comment:
was tempted to pass a `Row` here, but then `MetricSorter` has to build one from an array of strings.